test/parallel/test-crypto.js

```diff
@@ -260,7 +260,7 @@ assert.throws(function() {
 
 /**
```

Comment:
Is this line correct now?

Reply:
Yes it's correct: https://eslint.org/docs/rules/spaced-comment#always

Reply:
Is this some built-in exception for jsdoc?

Reply:
Yes. `'*'` is the default marker ([see here](https://github.com/eslint/eslint/blob/master/lib/rules/spaced-comment.js#L41-L44)), so `/**` can pass the check.